src/fragments/lib/project-setup/flutter/upgrade-guide/upgrade-guide.mdx

```diff
@@ -1,328 +1,173 @@
-With the release of v1 Amplify Flutter now supports Web and Desktop for Auth, API, Analytics, and Storage use cases. Developers can build cross-platform Flutter apps with Amplify that target iOS, Android, Web, and Desktop (macOS, Windows, Linux) using a single codebase.
+Amplify Flutter v2 has several changes that may require migration when upgrading from v1.
```

Comment:
Can we mention here and manage expectation on what kind of changes user should expect? E.g. "Amplify Flutter v2 has changes over Auth, Data and Storage to do xxx"